src/Sylius/Behat/Element/Admin/ProductOption/FormElement.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Admin\ProductOption;
+
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Sylius\Behat\Behaviour\ChecksCodeImmutability;
+use Sylius\Behat\Behaviour\NamesIt;
+use Sylius\Behat\Behaviour\SpecifiesItsField;
+use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
+use Webmozart\Assert\Assert;
+
+class FormElement extends BaseFormElement implements FormElementInterface
+{
+    use NamesIt;
+    use SpecifiesItsField;
+    use ChecksCodeImmutability;
+
+    public function addOptionValue(string $code, string $value): void
+    {
+        $count = count($this->getProductOptionValues());
+
+        $this->getElement('add_value')->click();
+        $this->getDocument()->waitFor(20, fn () => $count + 1 === count($this->getProductOptionValues()));
+
+        $optionValueForm = $this->getLastOptionValueElement();
+
+        $optionValueForm->fillField('Code', $code);
+
+        $optionValueForm->find('css', '[data-test-option-value-translation]')->setValue($value);
+    }
+
+    public function checkValidationMessageForOptionValues(string $message): bool
+    {
+        $optionValuesValidationElement = $this->getElement('values_validation')->find('css', '.sylius-validation-error');
+        if (null === $optionValuesValidationElement) {
+            throw new ElementNotFoundException($this->getDriver(), 'product option validation box', 'css', '.sylius-validation-error');
+        }
+
+        return $optionValuesValidationElement->getText() === $message;
+    }
+
+    public function nameItIn(string $name, string $language): void
+    {
+        $this->getDocument()->fillField(
+            sprintf('sylius_admin_product_option_translations_%s_name', $language),
+            $name,
+        );
+    }
+
+    public function isThereOptionValue(string $optionValue): bool
+    {
+        $optionValues = $this->getElement('values');
+
+        return $optionValues->has('css', sprintf('input[value="%s"]', $optionValue));
+    }
+
+    public function removeOptionValue(string $optionValue): void
+    {
+        if ($this->isThereOptionValue($optionValue)) {
+            $optionValues = $this->getElement('values');
+
+            $item = $optionValues
+                ->find('css', sprintf('div[data-form-collection="item"] input[value="%s"]', $optionValue))
+                ->getParent()
+                ->getParent()
+                ->getParent()
+                ->getParent()
+            ;
+            $item->clickLink('Delete');
```

Comment:
It would be good not to use translation here, but test attributes, ideally through `getElement` method if possible 🤔